docs/git/desktop/lesson.md

```diff
@@ -325,10 +325,62 @@ Try to use this format when making your own commits. GitHub Desktop tries to enc
 
 Explaining _why_ we're making the change can help people in the future to understand why things look the way they do, and what's important not to change.
 
+
+## Personal Development - Introspection 
+
+**Goals , Expectations  & Contributions (10 - 15 mins) **
+
+To identify trainees goal(s) for being on the Full Stack CYF course
+To highlight the expectations CodeYourFuture has for it's trainees
+What do you contribute to the community?
+**
+What do Developers do?  (20 mins)**
+In this section we'll explore how a professional behaves and thus how we expect trainees to behave.
+
+**
+Exercise (Why Git is useful) **
+In groups of 5, Brainstorm together in small or large groups imagine all the day-to-day activities and behaviours of a professional software developer. Think about how Git is a useful tool and how it might help developers in their role.
+You might want to use a Miro Board
+
+Once we have we have a large list, invite them to think about whether they do all those things already
+Discuss how to change things so they begin to do more and more of those things highlighting that practice is the way to develop
+
+All trainees are to complete a SDOT analysis (20 mins) 
+
+https://docs.google.com/document/d/1MCujCtdSUmfrVGr5Vd9nstoNz0nXMUY1nZ002KMl1rk/edit#
+
+
+
+
+
 ## Coursework going forwards
 
 For the next four modules, most weeks your homework is going to be to clone a repository from GitHub, write some code, and make a Pull Request with the code.
 
+Personal Development Coursework
```

Comment:
Hi @ElizabethlawalCYF - I think this might sit better over in this page:

https://github.com/CodeYourFuture/syllabus/blob/master/docs/git/desktop/homework.md

So that it goes with the rest of the coursework :)